jetty-ee10/jetty-ee10-webapp/src/main/config/modules/ee10-deploy.mod

```diff
@@ -9,6 +9,12 @@ deployment
 [environment]
 ee10
 
+[before]
+ee9-deploy
+ee8-deploy
+core-deploy
+static-deploy
```

Comment:
Are you sure you need to list them all?  Will the ordering work if some modules in the order are not enabled?

Reply:
Pretty sure I need to list them all.
When enabling `ee11-deploy` and `static-deploy`, without the full list there is no way to know if `ee11-deploy` is _before_ or not.

Reply:
I just tried and confirmed that the whole list is necessary.